src/components/Form.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+
+const Form =({handleClick})=>{
+    const [userNum, setUserNum] = useState(null);
+    
+    const handleChange = (event)=>{
+        setUserNum(event.target.value) ; 
+    }
+    return(
+        <>
+        <h2>Have a flavor of the show before binging!</h2>
+        <form action="" onSubmit={(event) =>{handleClick(event,userNum)}}>
+            <input type="text" id="searchInput" placeholder="Search a show here" onChange={(event)=>{handleChange(event)}}/>      
+            <button type="submit"> Show me the fun facts!</button>
+
+        </form>
+        </>
+        
+    )
+}
+export default Form ; 
```

Comment:
When i leave the search bar blank and hit submit it returns a random show anyways - maybe implement error handling so that user can not submit an empty input

Reply:
just noticed that it brought back a show called "Null Peta" because of the useState(null)